client/components/course/Sidebar/Relationship.vue

```diff
@@ -74,8 +94,11 @@ export default {
       return get(activity, `refs.${this.type}`, []);
     },
     onRelationshipChanged(associations) {
+      const ref = Array.isArray(associations)
+        ? map(associations, 'id')
+        : [get(associations, 'id')];
```

Comment:
This becomes simple mapper once you ensure that associations is always an array.

Reply:
`associations` argument is emitted from multiselect. It will be an `Object` if `multiple=false`

Reply:
Sorry, I misunderstood your comment. It's refactored now